ModulePresentationsForCAP/gap/ModulePresentationFunctors.gi

```diff
@@ -158,3 +158,60 @@ InstallMethod( FunctorLessGeneratorsRight,
     return functor;
     
 end );
+
```

Comment:
The suggestion are done in this commit
https://github.com/kamalsaleh/CAP_project/commit/80d6cf2ab3feb6f1c972d14588c36e23913f032e